src/routers/handlers/email/checkAnswer.ts

```diff
@@ -0,0 +1,83 @@
+import {Request, Response} from "express";
+import {GenericHandler} from "../generic";
+import {Session} from "@companieshouse/node-session-handler";
+import {logger} from "../../../lib/Logger";
+import {
+    COMPANY_EMAIL,
+    COMPANY_NUMBER,
+    SUBMISSION_ID,
+    TRANSACTION_CLOSE_ERROR
+} from "../../../constants/app.const";
+import {EMAIL_CHANGE_EMAIL_ADDRESS_URL} from "../../../config";
+import {createRegisteredEmailAddressResource} from "../../../services/company/createRegisteredEmailAddressResource";
+import {closeTransaction} from "../../../services/transaction/transaction.service";
+
+export class CheckAnswerHandler extends GenericHandler {
+
+    constructor() {
+        super();
+    }
+
+    async get(req: Request, response: Response): Promise<Object> {
+        logger.info(`GET request to serve company confirm page`);
+
+        const session: Session = req.session as Session;
+        const companyEmail: string | undefined = session.getExtraData(COMPANY_EMAIL);
+
+        return {
+            companyEmail: companyEmail,
+            backUri: EMAIL_CHANGE_EMAIL_ADDRESS_URL,
+            signoutBanner: true,
+            userEmail: req.session?.data.signin_info?.user_profile?.email
+        };
+    }
+
+    async post(req: Request, response: Response): Promise<any> {
+        logger.info(`POST request to serve company confirm page`);
+
+        const session: Session = req.session as Session;
+        const companyEmail = req.session?.getExtraData(COMPANY_EMAIL);
+        const emailConfirmation: string | undefined = req.body.emailConfirmation;
+
+        if (emailConfirmation === undefined) {
+            return {
+                statementError: "You need to accept the registered email address statement",
```

Comment:
Can we put any error constants into `constants/app.const` for consistency rather that inlining in code?